src/components/organisms/DiscoverCard/index.tsx

```diff
@@ -15,21 +15,21 @@ const customStyles = makeStyles({
   scroll: {
     width: "840px",
     height: "284px",
-    "&:hover": {
-      boxShadow: "0 8px 16px 0 rgba(0,0,0,0.2)",
-    },
   },
 });
 
-const InvestmentCard = () => {
+const DiscoverCard = () => {
   const classes = customStyles();
 
   // eslint-disable-next-line react-hooks/rules-of-hooks
   const [bitcoin, setBitcoin] = React.useState<assetsProps[]>([]);
 
-  axios.get(`http://localhost:3000/discoverItems`).then((res) => {
-    setBitcoin(res.data);
-  });
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  React.useEffect(() => {
+    axios.get(`http://localhost:3000/discoverItems`).then((res) => {
+      setBitcoin(res.data);
```

Comment:
no service file? Good practice to have one to keep all API calls in a single place.